en/docs/Learn/Extensions/SAML2SSO/configuring-identity-server-as-idp-for-sso.md

```diff
@@ -14,7 +17,7 @@ The topics below explain the configurations.
 
 ## Sharing the user store
 
-Initially, configure your user store(s), if you have not done so already, by following the instructions in [Configuring User Stores](../../../../Administer/ProductAdministration/ManagingUsersAndRoles/admin-introduction-to-user-management/) . Thereafter, point both WSO2 IS and WSO2 API Manager to your user stores(s) using the instructions given below. You do this to make sure that a user who tries to log in to the Developer Portal or the Publisher is authorized. When a user tries to log in to either of the applications, s/he is redirected to the configured identity provider (WSO2 IS in this case) where s/he provides the login credentials to be authenticated. In addition to this, the user should also be authorized by the system as some user roles do not have permission to perform certain actions. For the purpose of authorization, the IS and API Manager needs to have a shared user store and user management database (by default, this is the H2 database in the `<API-M_HOME>/repository/conf/user-mgt.xml` file) where the user's role and permissions are stored.
+Initially, configure your user store(s), if you have not done so already, by following the instructions in [Configuring User Stores]({{base_path}}/Administer/ProductAdministration/ManagingUsersAndRoles/ManagingUserStores/introduction-to-userstores/) . Thereafter, point both WSO2 IS and WSO2 API Manager to your user stores(s) using the instructions given below. You do this to make sure that a user who tries to log in to the Developer Portal or the Publisher is authorized. When a user tries to log in to either of the applications, s/he is redirected to the configured identity provider (WSO2 IS in this case) where s/he provides the login credentials to be authenticated. In addition to this, the user should also be authorized by the system as some user roles do not have permission to perform certain actions. For the purpose of authorization, the IS and API Manager needs to have a shared user store and user management database (by default, this is the H2 database in the `<API-M_HOME>/repository/conf/user-mgt.xml` file) where the user's role and permissions are stored.
```

Comment:
Break these into separate instructions.
1) Configure your user store(s) (if  not done so already)  by following the instructions in [Configuring User Stores]({{base_path}}/Administer/ProductAdministration/ManagingUsersAndRoles/ManagingUserStores/introduction-to-userstores/) 
2) Point both WSO2 IS and WSO2 API Manager to your user stores(s) using the instructions given below. 
This  is required to make sure that a user who tries to log in to the API Developer Portal or the API Publisher is authorized. When a user tries to log in to either of the applications, s/he is redirected to the configured identity provider (WSO2 IS in this case) where s/he provides the login credentials to be authenticated.In addition to this, the user should also be authorized by the system as some user roles do not have permission to perform certain actions. 
---certain actions.  ?? such as? 
Hence, for the purpose of authorization, the IS and API Manager needs to have a shared user store and user management database (by default, this is the H2 database in the `<API-M_HOME>/repository/conf/user-mgt.xml` file) where the user's role and permissions are stored.

Reply:
Thanks. Fixed. https://github.com/wso2/docs-apim/pull/544/commits/8931fbcc92eead2bd0dee32572193db7652507df